doc/pages/keys.asciidoc

```diff
@@ -837,7 +837,9 @@ The following keys are recognized by this mode to help with editing
 These keys are used to cancel long-running operations, either inside
 Kakoune or outside it. Because they are intended as a safety mechanism
 when something goes wrong, these keys are handled very early on in
-Kakoune's input processing, and therefore cannot be remapped in any mode.
+Kakoune's input processing, so they can only be remapped by changing
```

Comment:
So this allows us to maybe make commands like `map global normal <c-c> ...`
throw an error if `cancel_key=<c-c>`, and point to that option.

Not sure if we want to do this; it would be a bit more user friendly on one
hand but also enforce an ordering of map/set-option commands in kakrc which
would be weird.

Reply:
Yeah, this sounds like it would introduce a little more complexity on both the user's and Kakoune's side.  I do wonder if maybe the pointer about "warning, these two behave differently" belongs near the top of `:doc mapping` rather than at the very bottom of `:doc keys`?